nx_altair/draw_altair.py

```diff
@@ -289,6 +402,9 @@ def draw_networkx(
     edgelist=None,
     node_size=300,
     node_color='red',
+    node_label=None,
```

Comment:
* `node_label_size` should be `font_size`
* `node_label_color` should be `font_color`
* `node_label` should be `labels`.